features/etc/feature.properties

```diff
@@ -0,0 +1,200 @@
+#
+# Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
# WSO2 Inc. licenses this file to you under the Apache License,
```
Inc -> LLC 

Please check and add the correct license

Reply:
Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/16/commits/0b171a59a8c453736a4336a50e5c4d33325006c6